corehq/apps/reports/templates/reports/async/project_health_dashboard.html

```diff
@@ -0,0 +1,195 @@
+{% extends report_base|default:"reports/async/default.html" %}
+
+{% load hq_shared_tags %}
+{% load report_tags %}
+{% load i18n %}
+
+{% block reportcontent %}
+<div class="row">
+    {% include "reports/project_health/partials/last_month_stats.html" %}
+    {% if request.user.is_superuser %}
+    <p>
+        <a href="{% url 'domain_internal_settings' domain %}">
+        {% trans "Set Performance Threshold" %}
+        </a>
+    </p>
+    {% endif %}
+    <div class="col-md-6">
+        <h2>{% trans "Active Users Trend" %}</h2>
+            <p class="lead">
+                {% trans "Proportion of users that are active (submitting at least one form) over time." %}
+            </p>
+        <div id="active_linechart"><svg height="300px"></svg></div>
+    </div>
+    <div class="col-md-6">
+        <h2>{% trans "High / Low Performing User Trends" %}</h2>
+            <p class="lead">
+                {% trans "How the counts of high / low performing users have changed in the last six months." %}
+            </p>
+        <div id="perform_chart"><svg height="300px"></svg></div>
+    </div>
+    {% include "reports/project_health/partials/users_tables.html"%}
+</div>
+{% endblock reportcontent %}
+
+{% block js-inline %} {{ block.super }}
+    <script type="text/template" id="js-template-loading-report">
+        <div class="report-loading-container">
+            <div class="report-loading">
+                <h4>{% trans "Loading Report" %}</h4>
+                <i class="fa fa-spin fa-spinner"></i>
+            </div>
+        </div>
+    </script>
```

Comment:
Is this script block still necessary?  I'd expect it to be covered by `{{ block.super }}`